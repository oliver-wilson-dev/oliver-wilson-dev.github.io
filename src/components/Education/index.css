.degreeTitle + span {
    background-color: rgba(255, 23, 68, 0.7);
    border-radius: 20px;
    display: inline-block;
    font-size: 16px;
    font-weight: 400;
    letter-spacing: 2px;
    margin-left: 5px;
    margin-top:5px;
    padding: 5px 15px;
    text-transform: uppercase;
}

.degreeClassification span {
    background-color: rgba(0, 200, 83, 0.7);
    border-radius: 20px;
    font-size: 16px;
    font-weight: 400;
    letter-spacing: 2px;
    margin-left: 5px;
    padding: 5px 15px;
    text-transform: uppercase;
}

.degreeTitle {
    display: inline-block;
    margin-bottom: 0;
    margin-right: 5px;
}

.universityTitle {
    margin-bottom: 0px;
}

.paragraphEmoji {
    margin-right: 10px;
    font-size: 30px;
}