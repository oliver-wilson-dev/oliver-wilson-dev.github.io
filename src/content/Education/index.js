import React from 'react';
import styles from '../index.css';

export default {
  title: 'Education',
  content:
  <div>
    <h3 className={styles.universityTitle}>The University of Kent</h3>
    <h4 className={styles.degreeTitle}>
      Bachelor of Science in
      {' '}
    </h4>
    <span>Computer Science</span>
    <h4 className={styles.degreeClassification}>
      Classification:
      {' '}
      <span>first</span>
    </h4>
    <p>
      <span className={styles.paragraphEmoji} role="img" aria-label="university emoji">ðŸŽ“</span>
          I studied at the University of Kent for three years (2014 -2017)
          and obtained a first class degree with honours in Computer Sciecne upon my graduation.
    </p>
    <p>
          During my studies I worked on a number of exciting front end projects and really enjoyed
          the creativity that developing software for the web afforded me.
    </p>
  </div>,
};
