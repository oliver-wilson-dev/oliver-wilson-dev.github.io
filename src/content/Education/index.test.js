import Education from './index';

describe('Education Section', () => {
  it('should render correctly', () => {
    expect(Education).toMatchSnapshot();
  });
});
