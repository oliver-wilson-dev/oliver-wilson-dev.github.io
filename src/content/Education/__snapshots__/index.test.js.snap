// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Education Section should render correctly 1`] = `
Object {
  "content": <div>
    <h3
      className="universityTitle"
    >
      The University of Kent
    </h3>
    <h4
      className="degreeTitle"
    >
      Bachelor of Science in
    </h4>
    <span>
      Computer Science
    </span>
    <h4
      className="degreeClassification"
    >
      Classification:
      <span>
        first
      </span>
    </h4>
    <p>
      <span
        aria-label="university emoji"
        className="paragraphEmoji"
        role="img"
      >
        ðŸŽ“
      </span>
      I studied at the University of Kent for three years (2014 -2017) and obtained a first class degree with honours in Computer Science upon my graduation.
    </p>
    <p>
      During my studies I worked on a number of exciting front end projects and really enjoyed the creativity that developing software for the web afforded me.
    </p>
  </div>,
  "title": "Education",
}
`;
